resources/assets/css/front/bootstrap.css

```diff
@@ -0,0 +1,39 @@
+@tailwind preflight;
+
+* {
+    position: relative;
+    box-sizing: inherit;
+    margin: 0;
+    padding: 0;
+    color: inherit;
+    font: inherit;
+
+    &:after,
+    &:before {
+        box-sizing: inherit;
+    }
+}
+
+html {
+    box-sizing: border-box;
+    font-size: 16px;
```

Comment:
Hmm not sure if this one's a good idea for accessibility https://www.sitepoint.com/stop-maiming-bodies-the-perils-of-pixel-font-size/

Reply:
You're right, removed it